__tests__/app.test.js

```diff
@@ -122,6 +122,7 @@ describe("/api", () => {
               votes: 100,
               article_img_url:
                 "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              comment_count: 1
```

Comment:
Well done for changing this test to a flexible test - I would not add the comment count to the original test but do a fresh one for it